modules/auxiliary/scanner/http/wordpress_scanner.rb

```diff
@@ -126,7 +128,37 @@ def run_host(target_host)
         end
         print_status("#{target_host} - Finished scanning plugins")
       end
-      print_status("#{target_host} - Finished all scans")
+      #      if 1==1
+      if datastore['USERS']
+        print_status("#{target_host} - Searching Users")
+        url = wordpress_url_rest_api + '/users'
+        res = send_request_cgi({
+          'method' => 'GET',
+          'uri' => url
+        })
+        if res.nil?
+          print_error('Error getting response.')
+        elsif res.code == 200
+          parsed = JSON.parse(res.body)
+          parsed.map do |child|
+            name = child['name']
+            slug = child['slug']
+            print_good("#{target_host} - Detected user: #{name} Slug: #{slug}")
+            report_note(
```

Comment:
Instead of using `report_note` this should probably be reporting the username as a credential object with no password. That would enable users to then use the DB_ALL_USER option. You should leave the password blank (or nil) and mark it's status as untried.

Reply:
agree with @smcintyre-r7